quilt/tools/command.py

```diff
@@ -160,6 +161,19 @@ def _parse_package(name):
 
     return owner, pkg
 
+def _open_url(url):
+    try:
+        if sys.platform == 'win32':
+            os.startfile(url)
+        elif sys.platform == 'darwin':
+            with open(os.devnull, 'r+') as null:
+                subprocess.check_call(['open', url], stdin=null, stdout=null, stderr=null)
+        else:
+            with open(os.devnull, 'r+') as null:
+                subprocess.check_call(['xdg-open', url], stdin=null, stdout=null, stderr=null)
+    except (OSError, FileNotFoundError, subprocess.SubprocessError) as ex:
```

Comment:
You probably mean `Exception`. Sure. (Linker complains about that, but whatever.)